vertical-pod-autoscaler/pkg/recommender/checkpoint/checkpoint_writer.go

```diff
@@ -147,10 +147,4 @@ func subtractCurrentContainerMemoryPeak(a *model.AggregateContainerState, contai
 	if now.Before(container.MemoryWindowEnd) {
 		a.AggregateMemoryPeaks.SubtractSample(model.BytesFromMemoryAmount(container.GetMaxMemoryPeak()), 1.0, container.MemoryWindowEnd)
 	}
-	if now.Before(container.RSSWindowEnd) {
-		a.AggregateRSSPeaks.SubtractSample(model.BytesFromMemoryAmount(container.GetMaxRSSPeak()), 1.0, container.RSSWindowEnd)
-	}
-	if now.Before(container.JVMHeapCommittedWindowEnd) {
-		a.AggregateJVMHeapCommittedPeaks.SubtractSample(model.BytesFromMemoryAmount(container.GetMaxJVMHeapCommittedPeak()), 1.0, container.JVMHeapCommittedWindowEnd)
-	}
```

Comment:
excited to see this gone!

Reply:
nit: add note that we don't need this anymore with the binary decaying histogram

Reply:
Do you mean a comment in checkpoint writer?

Reply:
added comment